docs/New-ExternalHelp.md

```diff
@@ -5,82 +5,74 @@ schema: 2.0.0
+
 ## SYNTAX
 
 ```
 New-ExternalHelp -Path <String[]> -OutputPath <String> [-Encoding <Encoding>] [-Force] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-Create External help file from platyPS markdown.
+The **New-ExternalHelp** cmdlet creates an external help file based on markdown help files supported by PlatyPS. You can ship this with a module to provide help by using the  [Get-Help](https://msdn.microsoft.com/en-us/library/dd878343.aspx) cmdlet.
 
-You will get error messages if the markdown files do not follow the schema described in
-[platyPS.schema.md](https://github.com/PowerShell/platyPS/blob/master/platyPS.schema.md).
+If the markdown files that you specify do not follow the PlatyPS [Schema](https://github.com/PowerShell/platyPS/blob/master/platyPS.schema.md), this cmdlet returns error messages.
 ## EXAMPLES
 
-### Example 1 (Markdown folder)
+### Example 1: Create external help based on the contents of a folder
 ```
-PS C:\> New-ExternalHelp -Path .\docs -OutputPath out\platyPS\en-US
+PS C:\> New-ExternalHelp -Path ".\docs" -OutputPath "out\platyPS\en-US"
 
-    Directory: D:\dev\platyPS\out\platyPS\en-US
+    Directory: D:\Working\PlatyPS\out\platyPS\en-US
 
 
 Mode                LastWriteTime         Length Name
 ----                -------------         ------ ----
 -a----        5/19/2016  12:32 PM          46776 platyPS-help.xml
 ```
 
-Create external help file in output path directory.
-Note that directory should include language name.
-### Example 1 (With -Force and custom encoding)
+This command creates an external help file in the specified location. This command uses the best practice that the folder name includes the locale.
+
+### Example 2: Create help that uses custom encoding
 ```
-PS C:\> New-ExternalHelp .\docs -OutputPath out\platyPS\en-US -Force -Encoding ([System.Text.Encoding]::Unicode)
+PS C:\> New-ExternalHelp -Path ".\docs" -OutputPath "out\PlatyPS\en-US" -Force -Encoding ([System.Text.Encoding]::Unicode)
 
 
-    Directory: D:\dev\platyPS\out\platyPS\en-US
+    Directory: D:\Working\PlatyPS\out\PlatyPS\en-US
 
 
 Mode                LastWriteTime         Length Name
 ----                -------------         ------ ----
 -a----        5/22/2016   6:34 PM         132942 platyPS-help.xml
 ```
 
-Create and overwrite existing external help file in output path directory.
-Use Unicode Encoding for output file.
+This command creates an external help file in the specified location. This command specifies the *Force* parameter, therefore, it overwrites an existing file. The command specifies Unicode encoding for the created file.
 ## PARAMETERS
 
 ### -OutputPath
-Path to a folder where you want to put your external help file(s).
-The name should end with a locale folder, i.e. ".\out\platyPS\en-US".
+Specifies the path of a folder where this cmdlet saves your external help file. The folder name should end with a locale folder, as in the following example: ```.\out\PlatyPS\en-US\```.
```

Comment:
It works too, but you need (at least) 3 backticks for inline **codeblock**
For a inline (in one line) you need just 1.